content/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests.md

```diff
@@ -25,7 +25,8 @@ To attach a file to an issue or pull request conversation, drag and drop it into
 
 {% endtip %}
 
-The maximum size for files is 25MB and the maximum size for images is 10MB.
+The maximum size for files is 25MB and the maximum size for images is 10MB.  
+Videos can be up to 100 MB in size if the repository is owned by a user or organization on a paid GitHub plan.
```

Comment:
When we add docs for a beta feature, we always include a note to let users know that the feature might change before it's fully released (for example, [Configuring your watch settings for an individual repository](https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/viewing-your-subscriptions#configuring-your-watch-settings-for-an-individual-repository)).

An added complication with docs for beta features, is that they are usually available first on GitHub.com and later rolled out to other products. We're going to need to add a condition around your new content using a [liquid helper](https://github.com/github/docs/blob/main/contributing/liquid-helpers.md) so that it shows up only in the GitHub.com version of the article. 

Here, we need to add:

```suggestion
{% if currentVersion == "free-pro-team@latest" %}
Videos can be up to 100 MB in size if the repository is owned by a user or organization on a paid GitHub plan.

{% note %}

**Note:** Support for video attachments is currently in beta and subject to change. 

{% endnote %}

{% endif %}
```